data/strings/strings.txt

```diff
@@ -22241,7 +22241,9 @@
     mr = %d वस्तू
     nb = %d sted
     nl = %d plaats
-    pl = %d miejsce
+    pl:one = %d miejsce
```

Comment:
This word is used for tracks and bookmarks, so miejsce is not a correct translation. Also see https://github.com/organicmaps/organicmaps/pull/8245

Reply:
Should I wait till #8245 will be merged? Will there be conflicts?

Reply:
Don't wait, conflicts can be resolved.

Reply:
Changed to objects